les-nouvelles-de-l-archeologie.csl

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="fr-FR" page-range-format="expanded">
+  <info>
+    <title>Les nouvelles de l'archéologie (French)</title>
+    <title-short>NDA</title-short>
+    <id>http://www.zotero.org/styles/les-nouvelles-de-l-archeologie</id>
+    <link href="http://www.zotero.org/styles/les-nouvelles-de-l-archeologie" rel="self"/>
+    <link href="https://journals.openedition.org/nda/1554" rel="documentation"/>
+    <author>
+      <name>Sébastien Plutniak</name>
+      <email>sebastien.plutniak@posteo.net</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="history"/>
+    <issn>0242-7702</issn>
+    <eissn>2425-1941</eissn>
+    <summary>Style for the Les nouvelles de l'archéologie journal.</summary>
+    <updated>2019-10-06T10:57:00+01:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="editor" form="short">
+        <single>(éd.)</single>
+        <multiple>(éds.)</multiple>
+      </term>
+      <term name="page" form="short">
+        <single/>
+        <multiple/>
+      </term>
+      <term name="no date">À paraître</term>
```

Comment:
I don't think I like this. Forthcoming and no date are two very different concepts.

Reply:
Ok, I changed it in the new version